src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.js

```diff
@@ -68,134 +86,80 @@ describe.skip('VAOS Component: CanceledAppointmentsList', () => {
+      localStartTime: startDate.format('YYYY-MM-DDTHH:mm:ss.000ZZ'),
       locationId: '983GC',
+      location: createMockFacilityByVersion({
+        id: '983GC',
+        name: 'Cheyenne VA Medical Center',
+        address: {
+          postalCode: '82001-5356',
+          city: 'Cheyenne',
+          state: 'WY',
+          line: ['2360 East Pershing Boulevard'],
+        },
+        phone: '970-224-1550',
+      }),
     };
-    const appointment = createMockAppointmentByVersion({
-      version: 0,
-      ...data,
-    });
 
-    mockAppointmentInfo({ va: [appointment] });
-
-    mockFacilitiesFetchByVersion({
-      facilities: [
-        createMockFacilityByVersion({
-          id: '442GC',
-          name: 'Cheyenne VA Medical Center',
-          address: {
-            postalCode: '82001-5356',
-            city: 'Cheyenne',
-            state: 'WY',
-            line: ['2360 East Pershing Boulevard'],
-          },
-          phone: '307-778-7550',
-          version: 0,
-        }),
-      ],
-      version: 0,
-    });
+    mockFetchFutureAppointments(appointment);
 
+    // Act
     const screen = renderWithStoreAndRouter(<CanceledAppointmentsList />, {
       initialState,
-      reducers,
     });
 
-    await screen.findByText(
-      new RegExp(
-        moment()
-          .tz('America/Denver')
-          .format('dddd, MMMM D'),
-        'i',
-      ),
-    );
-
-    expect(
-      screen.getByText(
-        new RegExp(
-          moment()
-            .tz('America/Denver')
-            .format('h:mm'),
-          'i',
-        ),
-      ),
-    ).to.exist;
+    // Assert
+    await screen.findByText(new RegExp(startDate.format('dddd, MMMM D'), 'i'));
+    expect(screen.getByText(new RegExp(startDate.format('h:mm'), 'i'))).to
+      .exist;
     expect(await screen.findByText(/Cheyenne VA Medical Center/i)).to.exist;
     expect(screen.baseElement).not.to.contain.text('VA appointment');
     expect(screen.baseElement).to.contain.text('Canceled');
   });
 
   it('should not display when they have hidden statuses', () => {
-    const data = {
-      id: '1234',
-      status: APPOINTMENT_STATUS.noshow,
-      kind: 'clinic',
-      start: moment().format(),
-      locationId: '983GC',
-    };
-    const appointment = createMockAppointmentByVersion({
-      version: 0,
-      ...data,
-    });
+    // Arrange
+    const startDate = moment();
+    const appointment = mockBaseCanceledAppointment(startDate);
+    appointment.attributes.status = APPOINTMENT_STATUS.noshow;
+    mockFetchFutureAppointments(appointment);
 
-    mockAppointmentInfo({ va: [appointment] });
+    // Act
     const screen = renderWithStoreAndRouter(<CanceledAppointmentsList />, {
       initialState,
-      reducers,
     });
 
+    // Assert
     return expect(
       screen.findByText(/You don’t have any canceled appointments/i),
     ).to.eventually.be.ok;
   });
 
-  it('should not display when over 13 months away', () => {
```

Comment:
I removed this test since it overlapped in functionality with `'should not show canceled appointment if more than 395 days ahead'`